.github/workflows/release-gh-pages.yml

```diff
@@ -13,7 +13,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.PAT_GITHUB }}
+          persist-credentials: false
```

Comment:
I think my previous suggestion to disable persist credentials was wrong.
The checkout action should persist the PAT credentials so that the `git push` done later can use them.